tools-local/Microsoft.DotNet.Build.Tasks.Local/Microsoft.DotNet.Build.Tasks.Local.csproj

```diff
@@ -4,19 +4,41 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'net45'">.NETFramework,Version=v4.5</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort>netstandard1.5</NuGetTargetMonikerShort>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'net45'">.NETFramework,Version=v4.5.1</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'net45'">net451</NuGetTargetMonikerShort>
     <CLSCompliant>false</CLSCompliant>
     <ProjectGuid>{B507BED2-0A28-456A-99C6-AD7AA148E35F}</ProjectGuid>
     <OutputPath Condition="'$(TargetGroup)' != 'net45'">$(LocalToolRuntimePath)</OutputPath>
-    <IntermediateOutputPath Condition="'$(TargetGroup)' == 'net45'">$(IntermediateOutputPath)\net45</IntermediateOutputPath>
     <OutputPath Condition="'$(TargetGroup)' == 'net45'">$(LocalToolRuntimePath)\net46</OutputPath>
+    <IntermediateOutputPath>$(BaseIntermediateOutputPath)tools-local/$(MSBuildProjectName)/$(TargetGroup)</IntermediateOutputPath>
+    <PackageTargetFallback>dnxcore50;portable-net45+win8+wpa81</PackageTargetFallback>
+    <ContainsPackageReferences>true</ContainsPackageReferences>
+    <TargetFrameworkIdentifier>.NETStandard</TargetFrameworkIdentifier>
```

Comment:
This is a little bit deceiving because this doesn't actually control what framework we build for. I just set it to this to make sure the portable targets get imported in frameworktargeting.targets. We build the netfx flavor against nuget package references just like the netstandard flavor.